Include information on Kelvin in Wave 4 README (#35)
AdaGold/weather-report: ada-project-docs/wave-04.md

## LocationIQ and OpenWeather

In order to get the weather of the city, we will need to:
1. Get the latitude and longitude of the city using the [`LocationIQ`](https://locationiq.com/docs) API. 
2. Use the latitude and longitude response with the [`OpenWeather`](https://openweathermap.org/api/one-call-api) API to get current weather data. 

To get started and generate API tokens, create accounts and log into `LocationIQ` and `OpenWeather`.

### OpenWeather Response Format

3rd party APIs don't always provide data in a format that we can use directly. Since we can't control the formats that others provide, we'll often need to convert data from outside API responses.

> The OpenWeather API will return temperatures in Kelvin units rather than Farenheit or Celcius, which means we need to convert that temperature value ourselves. 

Reviewer: ```suggestion
The OpenWeather API will return temperatures in Kelvin units rather than Fahrenheit or Celsius, which means we need to convert that temperature value ourselves. 
```

> The formula for converting Kelvin into Celcius is:

Author: ```suggestion
The formula for converting Kelvin into Celsius is:
```

```
C = K - 273.15

> - `C` is the result in Celcius

Reviewer: ```suggestion
- `C` is the result in Celsius
```

- `K` is the original temperature in Kelvin
```

> The formula for converting Kelvin into Farenheit is:

Reviewer: ```suggestion
The formula for converting Kelvin into Fahrenheit is:
```

```
F = (K - 273.15) * (9 / 5) + 32

> - `F` is the result in Farenheit

Reviewer: ```suggestion
- `F` is the result in Fahrenheit
```
